flow/layers/layers.go

```diff
@@ -28,9 +82,28 @@ type LayerDHCPv4 struct {
 // LayerDNS wrapper to generate extra layer
 //proteus:generate
 type LayerDNS struct {
-	*layers.DNS
-	DNSQuestions []string
-	DNSAnswers   []string
+	ID     uint16
```

Comment:
I would prefer to write all this thing in protobuf directly instead of using proteus and then protobuf

Reply:
done